### DISCUSSION TITLE
Is it possible to calculate the flow field directly without outputting the mesh after the mesh is deformed?

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#2258] [Q&A]
Hello,
  I'm using SU2 for DOE sampling right now.  My current process is to change the values of the relevant variables in FFD, then use the `SU2_DEF` module to deform the mesh, it will output the mesh. Then I use `SU2_CFD` read mesh to calculate.

My question is, is it possible to deform the mesh without exporting the mesh (keep it in the computer memory) and then use `SU2_CFD` directly to perform the calculation, eliminating the need to export and read the mesh.

Thanks!

**Comment by pcarruscag** [comment]
You can have a look at the python wrapper examples and see if you can adapt something.

**Reply by Zcaic** [reply]
Okay, I will give it a try. Thank you!
